packages/components/tag/src/types.ts

```diff
@@ -0,0 +1,35 @@
+import type { DefineComponent } from 'vue'
+
+interface TagOriginalProps {
+  closable?: boolean
+  icon: string
+  color: string
+  checked: boolean
+  checkAble: boolean
+  isRound: boolean
+}
+
+export const PresetStatusColorTypes = ['success', 'processing', 'error', 'default', 'warning']
```

Comment:
```suggestion
export const PresetStatusColorTypes = ['pending', 'success',  'warning',  'error']
```
这个也移到 color.ts 里吧 processing 改成 pending
另外在 default.less 的 `@info-color: @primary-color;` 这一行后面加一个 `@pending-color: @primary-color;`